bob-grpc/proto/bob.proto

```diff
@@ -89,5 +89,8 @@ message BobError {
 
 // Delete operation options
 message DeleteOptions {
-    bool force_node = 1; // Force operation to be served by node to which it comes
+    repeated string remote_nodes = 1;
```

Comment:
Actually, we need this list only when `force_delete` is true. We also need to determine whether the request for deletion goes to aliens or not and we cannot do that by `remote_nodes` length check. So I think, we need a separate flag `is_alien`. `force_delete` flag can be removed and be estimated from `remote_nodes` length:

```proto
message DeleteOptions {
    bool force_node = 1; // Force operation to be served by node to which it comes
    bool is_alien = 2; // Shows if the request is for aliens
    repeated string force_remote_nodes = 3; // List of nodes in aliens to perform force operation (skip presence check) 
}
```